invenio_cli/helpers/scripts.py

```diff
@@ -153,6 +154,26 @@ def symlink_templates_folder(log_config):
     _force_symlink(target_path, link_path)
 
 
+def update_theme(local, log_config, docker_helper=None):
```

Comment:
Moderate: Not sure if it might be worth since the code might completely change after @fenekku re-working of the CLI. But the code of `update_statics` and `update_theme` functions are quite similar. Could it be done generic just adding a one or two more parameters (e.g. file to search `src_file`).

Reply:
@ppanero I totally agree with your proposal. I didn't spend more time on how this function would look like as I was thinking too that we will refactor the cli. I just duplicated the code I needed so that it would be easy to be refactored after :)

Reply:
My thinking is that I will put the theme part inside `update_statics` and probably not have an update container version. This is fine for now since like @zzacharo said, it will be easy to refactor.